packages/server/graphql/public/typeDefs/timelineEventTeamPromptComplete.graphql

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
Hmm the `lowerCamelCase` is consistent with other files in this directory.  I'm open to changing the convention (especially since the SDL-driven pattern is still pretty new), but I'd strongly prefer to do that in a separate PR rather than create inconsistency or change all the filenames here.

Reply:
The other files top level type is a mutation starting with a lower case letter. You have an object top level which starts with an uppercase letter.

Reply:
Ahh I see.  I'll go ahead and make this change